application/src/main/resources/org/opentripplanner/apis/gtfs/schema.graphqls

```diff
@@ -901,6 +901,8 @@ type Pattern implements Node {
   semanticHash: String
   "List of stops served by this pattern"
   stops: [Stop!]
+  "List of stops with pickup / dropoff type served by this pattern"
+  stopsInPattern: [StopInPattern!]!
```

Comment:
What do you need this for? Is it just for the index? Isn't it straightforward to compute the index yourself, first item in the list is 0 and so on?